src/NUnit3TestAdapterInstallV2/source.extension.vsixmanifest

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="NUnit.NUnit3TestAdapter" Version="3.5.1" Language="en-US" Publisher="Terje Sandstrom" />
+    <DisplayName>NUnit 3 Test Adapter</DisplayName>
+    <Description xml:space="preserve">NUnit 3 adapter for running tests in Visual Studio. Works with NUnit 3.x, use the NUnit 2 adapter for 2.x tests.</Description>
+    <MoreInfo>https://github.com/nunit/docs/wiki/VS-Adapter</MoreInfo>
+    <License>license.rtf</License>
+    <Icon>nunit3_32x32.png</Icon>
+    <PreviewImage>preview.png</PreviewImage>
+    <Tags>test, Testing, adapter, NUNit, TDD, NUnit3</Tags>
+  </Metadata>
+  <Installation>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[11.0,15.0)" />
```

Comment:
I am pretty sure this is a lowest SKU and above entry, so this will work on Community and every other edition.